### PR TITLE
Homepage Specs

### PR DESCRIPTION
**Pull request by ghost**
Issue #70

### CODE REVIEW

**Comment by ghost** [comment]
CONTRIBUTING and README should have no changes, yes?

File: doc/specs/homepage.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+# Homepage
+
+## Table of contents
+
+1. Motiviation
+1. Business rules
+1. Use cases
+1. Data model
```
Comment (ghost) [inline doc/specs/homepage.md]: All of these are number 1.

Also, data model should be removed.

**Comment by sneakysneakysneaky** [comment]
Must have had a dirty merge somewhere.... rebased.

File: doc/specs/homepage.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
+# Homepage
+
+## Table of contents
+
+1. Motiviation
+2. Business rules
+3. Use cases
+4. Data model
```
Comment (ghost) [inline doc/specs/homepage.md]: Data model is left here.

**Comment by ghost** [comment]
Left a comment, otherwise +1.

**Comment by sneakysneakysneaky** [comment]
Squashed & merged

**Pull request closed without merging**